test/asyncio_tests/test_asyncio_cursor.py

```diff
@@ -467,16 +467,24 @@ async def test_exhaust(self):
         sock = one(socks)
         cur = client[self.db.name].test.find(
             cursor_type=CursorType.EXHAUST).batch_size(1)
-        has_next = await cur.fetch_next
-        self.assertTrue(has_next)
+        await cur.fetch_next
+        self.assertTrue(cur.next_object())
```

Comment:
Can't we write this test in a way that it doesn't use these deprecated methods? We really should be testing this with `async for`

Reply:
Yes we could but I think that's out of scope for this ticket.

Reply:
It is? I was just suggesting not adding any new test code that uses deprecated methods, but OK.

Reply:
This test is already using fetch_next/next_object. Using async-for would be out of place and rewriting the whole test to use async-for is a much larger change, something that should happen in MOTOR-605.